test/integration/IntegrationTest.cpp

```diff
@@ -1284,6 +1298,210 @@ TEST_F(IntegrationTest, plain)
     }
 }
 
+TEST_F(IntegrationTest, audioOnlyNoPadding)
+{
+    if (__has_feature(address_sanitizer) || __has_feature(thread_sanitizer))
```

Comment:
Not sure why we are ignoring the tests when address sanitizer and thread sanitazer are enabled. But it seems a common patter in all tests in `IntegrationTest`. If the reason is we don't to want to spend the time running the `IntegrationTest` in all build flavours I wold suggest to change on `cmake` level. 
Only include IntegrationTest.cpp when `${CMAKE_BUILD_TYPE}` is not `DCHECK` nor `TCHECK` so we can get rid of this boilerplate condition